app/javascript/dashboard/routes/dashboard/helpcenter/composables/usePortal.js

```diff
@@ -0,0 +1,53 @@
+import { frontendURL } from 'dashboard/helper/URLHelper';
+import allLocales from 'shared/constants/locales.js';
+import { useMapGetter } from 'dashboard/composables/store';
+import { computed } from 'vue';
+
+/**
+ * COmposable to work with portal-related data and URLs.
+ *
+ * @returns {Object} The portal-related data and utilities.
+ */
+export const usePortal = () => {
+  /**
+   * Gets the current account ID.
+   * @type {number}
+   */
+  const accountId = useMapGetter('getCurrentAccountId');
+
+  /**
+   * Computes the current portal slug.
+   * @type {import('vue').ComputedRef<string>}
+   */
+  const portalSlug = computed(() => useMapGetter('getCurrentPortalSlug'));
+
+  /**
+   * Computes the current locale.
+   * @type {import('vue').ComputedRef<string>}
+   */
+  const locale = computed(() => useMapGetter('getCurrentLocale'));
```

Comment:
```suggestion
  const locale = useMapGetter('getCurrentLocale');
```

`useMapGetter` always returns a computed